speech/grpc/transcribe_streaming.py

```diff
@@ -208,7 +212,11 @@ def main():
             make_channel('speech.googleapis.com', 443)) as service:
         # For streaming audio from the microphone, there are three threads.
         # First, a thread that collects audio data as it comes in
```

Comment:
This comment should be right above the `with` block, since that's what it's describing.